apps/ledger-live-desktop/src/renderer/init.tsx

```diff
@@ -47,8 +47,9 @@ import { addDevice, removeDevice, resetDevices } from "~/renderer/actions/device
 import { Device } from "@ledgerhq/live-common/hw/actions/types";
 import { listCachedCurrencyIds } from "./bridge/cache";
 import { LogEntry } from "winston";
-import { importTrustchainStoreState } from "@ledgerhq/trustchain/store";
 import { importMarketState } from "./actions/market";
+import { fetchWallet } from "./actions/wallet";
+import { fetchTrustchain } from "./reducers/trustchain";
```

Comment:
Is there a reason for the `fetchWallet` to be in an `actions` file and the `fetchTrustchain` to be in a `reducers` one ?

Reply:
fetchTrustchain should probably move to a `actions/trustchain`, this file just don't exist yet.